### PR TITLE
[chore] Activate the log out button

### PR DESCRIPTION
**Pull request by wanjikum**
# JIRA TICKET NAME:
[CB-124](https://issues.openmrs.org/browse/CB-124)
Activate the log-out button

## SUMMARY:
Activated the Log out button in the reference application and standalone version.

### CODE REVIEW

File: app/js/components/common/Header.js [hunk 1]
```diff
@@ -11,9 +11,13 @@ export class Header extends Component {
       locationTags: [],
       currentLocationTag: "",
       defaultLocation: "",
-      currentUser: ""
+      currentLogOutUrl: "empty",
+      currentwindowUrl: window.location.href.includes("modules-refapp") ?
+        '/openmrs/appui/header/logout.action?successUrl=openmrs':
+        '/openmrs-standalone/appui/header/logout.action?successUrl=openmrs-standalone',
     };
```
Comment (mseaton) [inline app/js/components/common/Header.js]: We cannot hardcode /openmrs/... or /openmrs-standalone.../ into here.  @djazayeri or @rkorytkowski - can you comment on the proper way to get the context path?

Comment (wanjikum) [reply app/js/components/common/Header.js]: thanks @mseaton, I will work on it

Comment (wanjikum) [reply app/js/components/common/Header.js]: @djazayeri, @rkorytkowski Please help me out.

Comment (wanjikum) [reply app/js/components/common/Header.js]: @mseaton Is it okay now?

Comment (dkayiwa) [reply app/js/components/common/Header.js]: @wanjikum did you test and confirm that it is working well?

**Comment by dkayiwa** [comment]
@wanjikum did you test and confirm that it is working well?

**Comment by wanjikum** [comment]
@dkayiwa yes, It is working well.

**Comment by wanjikum** [comment]
@dkayiwa I have uploaded the changes to the modules ref app.

**Pull request merged**